Week1/homework/exercise5.js

```diff
@@ -0,0 +1,14 @@
+const z = 7.25;
+console.log (z);
+const a = 7;
+console.log (a) ;
+if (z > a)
+    
+{
+        const b = z
+        console.log(b)
+}
```

Comment:
👍 but there seems to be an indentation problem here with some extra spacing and new lines which aren't required. 🙂